test/query_users.js

```diff
@@ -73,6 +73,16 @@ describe('Query Users', function() {
 				{ username: { $autocomplete: 'ro' } },
 			],
 		});
+
+		// the users are not always returned in the same order, sort 'em
+		response.users.sort((a, b) => {
```

Comment:
This is fine for tests to pass but not the actual solution.

We should fix it adapting sort parameters so that server returns a deterministic order or fix the issue in server.